src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -599,7 +599,7 @@ public function orWhereBelongsTo($related, $relationshipName = null)
      * Add subselect queries to include an aggregate value for a relationship.
      *
      * @param  mixed  $relations
-     * @param  string  $column
+     * @param  string|\Illuminate\Contracts\Database\Query\Expression  $column
```

Comment:
We usually put the object before the scalar.
```suggestion
     * @param \Illuminate\Contracts\Database\Query\Expression|string  $column
```